keanu-python/keanu/model.py

```diff
@@ -1,27 +1,28 @@
 from .net import BayesNet
 from .vertex.base import Vertex
+from typing import Any, Mapping
 
 class Model:
-    def __init__(self, vertices={}):
+    def __init__(self, vertices: Mapping[str, Vertex] = {}) -> None:
         self.__dict__["_vertices"] = {}
         self.__dict__["_vertices"].update(vertices)
 
-    def to_bayes_net(self):
+    def to_bayes_net(self) -> BayesNet:
         return BayesNet((filter(lambda vertex: isinstance(vertex, Vertex), self._vertices.values())))
 
-    def __setattr__(self, k, v):
+    def __setattr__(self, k: str, v: Vertex) -> None:
         if k in self.__dict__:
             super(Model, self).__setattr__(k, v)
         else:
             self._vertices[k] = v
 
-    def __getattr__(self, k):
+    def __getattr__(self, k: str) -> Vertex:
         if k in self.__dict__:
             return self.__dict__[k]
         return self._vertices[k]
 
-    def __enter__(self, *args, **kwargs):
+    def __enter__(self, *args: Any, **kwargs: Any) -> 'Model':
```

Comment:
Why's this `Model` in quotes?

Reply:
Oh does that just signify it's that class

Reply:
Without quotes, it causes a runtime error